swarm/pss/pss.go

```diff
@@ -518,6 +518,29 @@ func (self *Pss) GetSymmetricKey(symkeyid string) ([]byte, error) {
 	return symkey, nil
 }
 
+// Returns all recorded topic and address combination for a specific public key
+func (self *Pss) GetPeer(keyid string) (topic []Topic, address []PssAddress, err error) {
```

Comment:
If you renamed `api.GetPeerTopics` imho this should be renamed too (this does not return the peer neither)